config/i18n-tasks.yml

```diff
@@ -115,6 +116,7 @@ ignore_unused:
 - 'availability_component.*'
 - 'errors.messages.*'
 - 'size.{billion,million,thousand,unit}'
+- 'date.formats.month_year' # l helper is not captured
```

Comment:
If the `l` helper isn't captures, what's the benefit of using it over `#to_formatted_s`?

Reply:
The 'l' helper will provide localized content. For this case, while in english it would say `February 2022`, in portuguese it would say `Fevereiro 2022`. This helper also allows me to use `Fevereiro / 2022`, which is the most common way to present dates in pt-BR

Reply:
Gotcha, thanks!